File: module-music/src/main/java/com/yyxnb/module_music/ui/MusicLockActivity.java
package com.yyxnb.module_music.ui;

import android.os.Bundle;

import com.yyxnb.common_base.base.BaseActivity;
import com.yyxnb.what.music.interfaces.MusicPlayerEventListener;
import com.yyxnb.module_music.R;
import com.yyxnb.module_music.bean.MusicBean;

/**
 * 音乐锁屏播放
 */
public class MusicLockActivity extends BaseActivity implements MusicPlayerEventListener<MusicBean> {


    @Override
    public int initLayoutResId() {
        return R.layout.activity_music_lock;
    }

    @Override
    public void initView(Bundle savedInstanceState) {

    }

    @Override
    public void onMusicPlayerState(int playerState, String message) {

    }

    @Override
    public void onPrepared(long totalDuration) {

    }

    @Override
    public void onBufferingUpdate(int percent) {

    }

    @Override
    public void onInfo(int event, int extra) {

    }

    @Override
    public void onPlayMusicOnInfo(MusicBean musicInfo, int position) {

    }

    @Override
    public void onMusicPathInvalid(MusicBean musicInfo, int position) {

    }

    @Override
    public void onTaskRuntime(long totalDuration, long currentDuration, long alarmResidueDuration, int bufferProgress) {

    }

    @Override
    public void onPlayerConfig(int playModel, int alarmModel, boolean isToast) {

    }

}

File: module_music/src/main/java/com/yyxnb/module_music/ui/MusicLockActivity.kt
package com.yyxnb.module_music.ui

import android.os.Bundle
import com.yyxnb.common_base.base.BaseActivity
import com.yyxnb.lib_music.interfaces.MusicPlayerEventListener
import com.yyxnb.module_music.R
import com.yyxnb.module_music.bean.MusicBean

/**
 * 音乐锁屏播放
 */
class MusicLockActivity : BaseActivity(), MusicPlayerEventListener<MusicBean> {
    override fun initLayoutResId(): Int {
        return R.layout.activity_music_lock
    }

    override fun initView(savedInstanceState: Bundle?) {}
    override fun onMusicPlayerState(playerState: Int, message: String) {}
    override fun onPrepared(totalDuration: Long) {}
    override fun onBufferingUpdate(percent: Int) {}
    override fun onInfo(event: Int, extra: Int) {}
    override fun onPlayMusicOnInfo(musicInfo: MusicBean, position: Int) {}
    override fun onMusicPathInvalid(musicInfo: MusicBean, position: Int) {}
    override fun onTaskRuntime(totalDuration: Long, currentDuration: Long, alarmResidueDuration: Long, bufferProgress: Int) {}
    override fun onPlayerConfig(playModel: Int, alarmModel: Int, isToast: Boolean) {}
}